### DISCUSSION TITLE
Document examples of default pragma arguments

### DISCUSSION THREAD

**Discussion by nipuna-perera** [dagger/dagger#12181] [Help]
### What is the issue?

More context [here](https://discord.com/channels/707636530424053791/1123687185837740053/1304085738824011777). 

Some defaults are not intuitive. It's better to be documented so devs can look for it if it errors. The biggest culprit is the slice of string. 

This works.
```go
// +default=["one","two","three"]
```

This doesn't (Still valid json)
```go
// +default="['one','two','three']"
```

This documentation should ideally be here - https://docs.dagger.io/api/arguments/#default-values

**Comment by jedevc** [comment]
> This doesn't (Still valid json)
> 
> ```go
> // +default="['one','two','three']"
> ```

This is a valid JSON string, not a valid JSON list (also note, even the contents of the JSON string aren't valid JSON - only double quotes are valid JSON strings).

Agreed this should be better documented, but I don't think we should make the second example work.
